src/containers/course-sections-list/CourseSectionsList.tsx

```diff
@@ -13,16 +13,20 @@ import DragIndicatorIcon from '@mui/icons-material/DragIndicator'
 import Divider from '@mui/material/Divider'
 import MenuItem from '@mui/material/MenuItem'
 import Typography from '@mui/material/Typography'
+import Crop75Icon from '@mui/icons-material/Crop75'
```

Comment:
think about import `addActivityMenuItems` from `constans` and also necessary imports for it